src/fetchers/stats-fetcher.js

```diff
@@ -241,7 +241,7 @@ const fetchStats = async (
     stats.totalCommits = await totalCommitsFetcher(username);
   }
 
-  // if count_private then add private commits to totalCommits so far.
+  // if count_private then add private contributions to totalCommits so far.
   if (count_private) {
     stats.totalCommits +=
```

Comment:
Of course, updating the documentation is important, but I think it would be better if we could update the actual logic in the code. Adding private contributions to totalContributions instead of totalCommits seems much more logical. @rickstaa what do you think?

Reply:
That would make much more sense to me, in which case would the `count_private` option still be needed?

Reply:
Yes, the count_private option would still make sense, although the question of whether it's needed is up to @rickstaa and @anuraghazra. I always thought it was unnecessary since I can't think of why someone would want to disable the option (more stats = better), but you'll have to ask them for their opinion before removing the option altogether. Feel free to make the change from totalContributions to totalCommits though, and I'll approve your PR!

Reply:
What would the `count_private` option do? Would it just add private contributions to total commits still? (If so, when would anyone want that behaviour, as it seems like quite a misleading stat?) More than happy to leave it, just not quite sure what its purpose is.

I don't think there's a stat displayed anywhere for total contributions yet, I can only see one for total number of repos contributed to - but I might have overlooked it.

I'm not confident at all in my PR, just wanted to have somewhere to start/continue the discussion, no worries about approvals for now.

Reply:
There is a field called totalContributions, which shows total number of commits + prs + issues, etc. The count_private option would show totalContributions with private repos included instead of counting contributions only from public repos. Currently the option adds number of private contributions to total number of commits; instead, it should add number of private contributions to total number of _contributions_.

Reply:
Ah ok, would `totalContributions` be a new thing that would be displayed on the card? It doesn't seem as though the stat is tracked yet within GRS, [as far as I can tell](https://github.com/anuraghazra/github-readme-stats/blob/master/src/fetchers/stats-fetcher.js#L188-L196):

<img width="510" alt="Screenshot 2023-02-08 at 13 10 16" src="https://user-images.githubusercontent.com/35763376/217539515-08bc2008-b811-4eee-82e0-73a4f9fc2ace.png">

Reply:
`contributedTo` is what I meant. (The variable name is a bit ambiguous since everything else is in the form of `totalXXX`)

https://github.com/anuraghazra/github-readme-stats/blob/112000667c01f18fd161f204ae3ee796ec2e3011/src/fetchers/stats-fetcher.js#L245-L251

Instead of adding the `restrictedContributionsCount` to `stats.totalCommits`, we should add it to `stats.ContributedTo`.

Reply:
I've made the change (https://github.com/anuraghazra/github-readme-stats/commit/c90c1945888a16ea4a098ac240de979fa54a0fa2) to save you some trouble. Hope you don't mind!

Reply:
I've also updated the documentation to reflect this change. I hope you're okay with me committing to your branch. Just wanted to save you the back-and-forth of review => requested changes => updated code => review :)